include/simdjson/common_defs.h

```diff
@@ -6,6 +6,9 @@
 
 namespace simdjson {
 
+// defined in src/to_chars
+char *to_chars(char *first, const char *last, double value);
```

Comment:
Is this intended to be public? If not, we probably want it in `internal` just to keep the namespace clear. Even when we mark things @private, text editors with autocomplete will not always honor that.

Reply:
Done. It is now hidden.